gendox-core-api/src/main/java/dev/ctrlspace/gendox/gendoxcoreapi/controller/ProjectController.java

```diff
@@ -48,12 +54,19 @@ private Page<Project> getAll(@Valid ProjectCriteria criteria, Pageable pageable)
     }
 
     // TODO: preauthorize has OP_CREATE_PROJECT for the requested organization
-    @PostMapping("/projects")
-    private Project create(@RequestBody Project project) throws GendoxException {
-        // TODO: validate that the user has access to the organization
-        // TODO: All Orginizations ADMINS + the creator of the project becomes members of the project
+    @PostMapping(value = "/projects", consumes = {"application/json"})
+    @ResponseStatus(value = HttpStatus.CREATED)
+    public Project createProject(@RequestBody ProjectDTO projectDTO, @Valid ProjectCriteria criteria) throws Exception {
```

Comment:
We dont need ProjectCriteria here. Whatever is the projectDTO needs to be store in the DB.
PreAuthorize will check if the user has permission to create project in the org specified in  projectDTO.orginizationId.